src/FINDOLOGIC/Export/Helpers/UsergroupAwareMultiValue.php

```diff
@@ -33,7 +33,12 @@ public function addValue(UsergroupAwareMultiValueItem $value)
 
     public function setAllValues($values)
     {
-        $this->values = $values;
+        $this->values = [];
+
+        /* @var UsergroupAwareMultiValueItem $value */
+        foreach ($values as $value) {
```

Comment:
Adding in a loop is super inefficient when we could just assign the input array. To achieve 100% coverage, I'd rather have a test that just gets the value.

Reply:
Forget what I said. I know realize that this fixes a bug.